FunctorCategories/gap/PreSheaves.gi

```diff
@@ -1243,7 +1243,7 @@ InstallMethodWithCache( PreSheavesOfFpEnrichedCategory,
             Add( PSh!.compiler_hints.category_attribute_names,
                  "AssociatedCategoryOfColimitQuiversOfSourceCategory" );
             
-            if IsCategoryOfSkeletalFinSets( C ) then
+            if IsCategoryOfSkeletalFinSets( C ) and IsCategoryOfSkeletalFinSets( H ) then
```

Comment:
Is the `H` here the same `H` as the one in `AddHomomorphismStructureOnObjects` etc. below? If yes, I think it should be renamed to `sFinSets` inside `AddHomomorphismStructureOnObjects` etc. If no, still one of the `H`s should be renamed to avoid confusion and programming errors.

Reply:
> Is the `H` here the same `H` as the one in `AddHomomorphismStructureOnObjects` etc. below?

Yes.

> If yes, I think it should be renamed to `sFinSets` inside `AddHomomorphismStructureOnObjects` etc. If no, still one of the `H`s should be renamed to avoid confusion and programming errors.

Together with Kamal and Felix, I am planning to generalize the code used in the Add-methods (e.g., `ExternalHomOnObjectsEqualizerDataUsingCoYonedaLemma`) to work for categories different from sFinSets. So I decided not to touch this code and just indicate with the above `IsCategoryOfSkeletalFinSets( H )` that we are not there yet.